main.py

```diff
@@ -11,6 +12,7 @@ def main():
     event_json = request.get_json()
     if event_warrants_label_check(event_json):
         pull_request = PullRequest(event_json)
+        sys.stdout.write("Checking labels for PR {}".format(pull_request.issue_url))
```

Comment:
You could have just gone with `print("Checking labels for PR {}".format(pull_request.issue_url))` for this as that writes directly to stdout.